src/crewai/telemetry/telemetry.py

```diff
@@ -253,16 +299,17 @@ def crew_execution_span(self, crew):
                             {
                                 "id": str(task.id),
                                 "description": task.description,
+                                "expected_output": task.expected_output,
                                 "async_execution?": task.async_execution,
-                                "output": task.expected_output,
+                                "human_input?": task.human_input,
                                 "agent_role": task.agent.role if task.agent else "None",
                                 "context": (
                                     [task.description for task in task.context]
                                     if task.context
-                                    else "None"
```

Comment:
Is this fine for you, @joaomdmoura?

Reply:
I think this might break clickhouse, let's keep it as a string, as the conversion don't know how to translate this to JSON iirc

Reply:
Just checked and it works fine. The final json has `"context": null`. I think it's better.